package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "orch-stubs",
+  "version": "1.0.0",
+  "description": "Performance and scaling stubs owned by Orchestration",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "sam build",
+    "start:local": "sam local start-api",
+    "check": "npm run check:lint && npm run check:prettier",
+    "check:lint": "eslint .",
+    "check:prettier": "prettier --check \"**/*.{ts,json}\"",
+    "fix": "npm run fix:lint && npm run fix:prettier",
+    "fix:lint": "eslint . --fix",
+    "fix:prettier": "prettier --write \"**/*.{ts,json}\"",
+    "test": "jest --passWithNoTests",
+    "test:coverage": "jest --coverage --passWithNoTests"
+  },
+  "prettier": {
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true
+  },
+  "author": "",
+  "dependencies": {
+    "esbuild": "^0.23.0"
+  },
+  "devDependencies": {
```

Comment:
I've looked into this a little. I don't think we need babel at, and can remove some of the others too. Let's chat about this

Reply:
I added it for supertest but if you've found a way to not use it, fab!